reproject/healpix/utils.py

```diff
@@ -49,7 +49,8 @@ def parse_input_healpix_data(input_data, field=0, hdu_in=None):
 
     if isinstance(input_data, six.string_types):
         from healpy import read_map
-        array_in, header = read_map(input_data, verbose=False, h=True, field=field, hdu=1 if hdu_in is None else hdu_in)
+        array_in, header = read_map(input_data, verbose=False, h=True, field=field,
+                                    hdu=1 if hdu_in is None else hdu_in)
```

Comment:
Would it be worth just defaulting to 1 in this function too and avoiding the 'if' logic here?